code/code/sys/configuration.h

```diff
@@ -1,7 +1,15 @@
 #pragma once
 
+#include <string>
+#include <string_view>
 #include "sstring.h"
 
+inline constexpr const char* MUD_NAME = "SneezyMUD";
+extern sstring MUD_NAME_VERS;
```

Comment:
Nit: Owned by main, but exported by configuration.

Reply:
Is it actually owned by main? In my mind it was owned by configuration and simply initialized in main out of necessity. It "feels" like it should be in `configuration.h` to me, but after looking at all the files that are including `configuration.h` and which of those are actually accessing those variables, there's probably a better place to declare them. Though I'm not sure where that would be, yet. Perhaps `db.h`?

Reply:
No idea, and not much of a preference. I'll leave it to you to pick a place for it.

Reply:
So after looking through this again, it still feels to me that `configuration.h` is the logical place to declare these variables. Most of the files that reference either of them already had `configuration.h` included, so it just requires adding the include to a couple extra files. Any objections?